app/controllers/rubric_item_assignments_controller.rb

```
@@ -0,0 +1,66 @@
+      score.score = assigned_points
+      score.save
+
+      flash[:success] = assignment.assigned ? "Rubric item assigned" : "Rubric item unassigned"
+
+      # Redirect back to the submission view
+    else
+      flash[:error] = "Failed to update rubric item assignment"
+    end
+    redirect_to view_course_assessment_submission_path(@course, @assessment, @submission,
+                                                       params[:header_position])
+  end
```

RuboCop (1.73)

[convention] Assignment Branch Condition size for toggle is too high. [<8, 31, 5> 32.4/23]

(Metrics/AbcSize)